poh/src/poh_recorder.rs

```diff
@@ -573,14 +573,14 @@ impl PohRecorder {
         self.leader_last_tick_height = leader_last_tick_height;
     }
 
-    pub fn set_bank(&mut self, bank: &Arc<Bank>, track_transaction_indexes: bool) {
+    pub fn set_bank(&mut self, bank: Arc<Bank>, track_transaction_indexes: bool) {
         assert!(self.working_bank.is_none());
-        self.leader_bank_notifier.set_in_progress(bank);
+        self.leader_bank_notifier.set_in_progress(&bank);
```

Comment:
hehe, `set_in_progress` seems to be a rare exception for this `Arc<_>` arg fixup saga... lol

Reply:
Yeah, I think the exception is justified as it does not take ownership of the Arc. It uses the arc to create a weak-reference.

(I know you know this, just making it clear for anyone else looking at the PR why this **should** be an exception)